docker/rootfs/etc/nginx/conf.d/include/force-ssl.conf

```diff
@@ -1,3 +1,14 @@
+# Since force-ssl.conf has now moved to the server section it overrides
+# the LetsEncrypt config which is inside a location section
+# Set FORCE variable in first two if tests pass and action in the third
+set $FORCE "";
 if ($scheme = "http") {
-	return 301 https://$host$request_uri;
+        set $FORCE 'H';
 }
+if ($request_uri !~ "^\/.well-known\/acme-challenge\/(.*)") {
```

Comment:
The dot (`.`) should probably be escaped in the regex:

```regex
^\/\.well-known\/acme-challenge\/(.*)
```

Reply:
Nice catch @lbogdan! 
Yes, it would match not only `.well-known/acme-challenge/` but also `xwell-known/acme-challenge/`, `1well-known/acme-challenge/`, `_well-known/acme-challenge/`, etc. 

@EDIflyer can you please push the change to the PR?

Reply:
OK I've pushed that update although I see the main develop branch has changed from `FORCE` to `TEST` - 

```nginx
set $test "";
if ($scheme = "http") {
	set $test "H";
}
if ($request_uri = /.well-known/acme-challenge/test-challenge) {
	set $test "${test}T";
}
if ($test = H) {
	return 301 https://$host$request_uri;
}
```
[docker/rootfs/etc/nginx/conf.d/include/force-ssl.conf](https://github.com/NginxProxyManager/nginx-proxy-manager/blob/develop/docker/rootfs/etc/nginx/conf.d/include/force-ssl.conf)

It is therefore blocking this merge and won't autogenerate a container.  Not sure if the underlying fix they have made resolved the issue or not?

Reply:
The underlying fix has not resolved the issue. The URI can change per test, so your code works. We could refactor the existing to:
```
set $test "";
if ($scheme = "http") {
	set $test "H";
}
if ($request_uri ~ "^\/\.well-known\/acme-challenge\/(.*)" {
	set $test "${test}T";
}
if ($test = H) {
	return 301 https://$host$request_uri;
}
```

Reply:
Thanks @willyouvba - have updated the commit accordingly.

Reply:
Sorry @EDIflyer, what I meant was: the contents of force-ssl.conf as it currently exists can be updated to the above (or modified completely to your version). Amalgamating the two won't work. It also looks like I lost a parenthesis at the end of the regex - I've updated my previous comment - sorry 😥.

Reply:
Ahh, that explains why things went down for me overnight when Watchtower pulled the new image then 😂. Will try to change it to your revised version.

Reply:
@willyouvba OK, can confirm sites back up and running using this new system!